packages/react/src/primitives/SelectField/SelectField.tsx

```diff
@@ -10,6 +10,31 @@ import { SelectFieldProps, Primitive } from '../types';
 import { splitPrimitiveProps } from '../shared/styleUtils';
 import { useStableId } from '../shared/utils';
 
+interface SelectFieldChildrenProps {
+  children?: React.ReactNode;
+  options?: SelectFieldProps['options'];
+}
+
+const selectFieldChildren = ({
+  children,
+  options,
+}: SelectFieldChildrenProps) => {
+  if (children) {
+    if (options?.length) {
```

Comment:
https://caniuse.com/mdn-javascript_operators_optional_chaining

Oh we can use javascript optional chaining??

![clap](https://media.giphy.com/media/3og0IuE1EjI5ZQzr3i/giphy-downsized.gif)

Reply:
Yeah, and I believe TypeScript should handle the conversion to ES5 if not.